pkg/operator/controller/certificate/controller.go

```diff
@@ -79,7 +79,7 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 		} else {
 			errs = append(errs, fmt.Errorf("failed to get clusteringress: %v", err))
 		}
-	} else {
+	} else if len(ingress.Status.Domain) > 0 {
```

Comment:
something about this feels off... should we have a higher level concept of accepted/rejected abstracted behind a function call or something?

Reply:
Will do.